TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
The 99% part was just a guess. Guess i was wrong :smile: 

I meant not "a big issue", like in "it won't hurt anything". It's not like your site will break because you do not use the "NodeTypes" package with the above TypoScript definition in place. In my eyes it's just a little help to get people started. Like we already have here:
https://github.com/neos/neos-development-collection/blob/master/TYPO3.Neos.Kickstarter/Resources/Private/Generator/Configuration/NodeTypes.yaml#L8

We could also take out the lines and just leave them in as a comment.